WalletWasabi.Fluent/Views/Wallets/Send/PrivacyWarningsView.axaml

```diff
@@ -163,7 +168,8 @@
           <Border Classes="warning">
             <DockPanel>
               <PathIcon />
-              <TextBlock Text="Transaction uses unconfirmed coins." />
+              <TextBlock Text="Transaction uses unconfirmed coins." 
+                         ToolTip.Tip="This may cause your transaction to be delayed or rejected and you may not pay the desired amount of mining fees. Wait for a confirmation to only use confirmed coins." />
```

Comment:
_rejected_: When you have incoming unconfirmed coins, there is no guarantee they are yours/will become yours. i.e. sending party might RBF/cancel.
_delayed_: afaik the fee estimation doesn't take it into account that a coin comes from a coinjoin? This will delay your tx.

> What are the benefits of waiting?

So (when confirmed) you know they are yours and fee estimations work correctly